src/Component/Header/style.scss

```
@@ -0,0 +1,214 @@
+            }
+        }
+    }
+    &_account {
+        @include Responsive(Mobile) {
+            width: 15%;
+        }
+
+        &_list {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            
+            .mobile-hide {
+                @include Responsive(Mobile) {
+                    display: none;
+                    @include paddingX(0,0);
+                }
+            }
+        }
+
+        &_item {
+            @include paddingX(1.4rem,1.4rem);
+            color: $blackColor;
+            font-size: 1.4rem;
+
+            svg {
+                transition: all .35s ease;
+                font-size: 1.4rem;
+                &:hover {
+                    color: $primaryColor;
+                }
+                @include Responsive(Mobile) {
+                    font-size: 2rem;
+                }
+            }
+            &:focus {
+                box-shadow: unset!important;
+            }
+            &:hover {
+                color: $primaryColor;
+            }
+        }
+    }
+
+    &_menu-toggle {
+        position: fixed;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        box-shadow: 0 4px 20px rgb(38 38 38 / 10%);
+        z-index: 1000;
+        height: 100vh;
+        width: 30rem;
+        transition: all .3s ease-in-out;
+
+        nav {
+            width: 30rem;
+            position: absolute;
+            left: 0;
+            top: 0;
+            bottom: 0;
+            border-right: 1px solid $whiteColor;
+            background-color: #262626;
+            color: $whiteColor;
+            overflow-y: auto;
+        }
+
+        &_close-btn{
+            font-size: 2rem;
+            line-height: 1.5;
+            margin-left: 1rem;
+            color: $whiteColor;
+            position: absolute;
+            right: 2rem;
+        }
+
+        &_logo {
+            margin: 1.5rem 0;
+            width: 100%;
+            font-weight: 300;
+            text-align: left;
+            transition: width .3s ease-in-out;
+            padding-left: 3.5rem;
+
+            span {
+                color: $primaryColor;
+                font-weight: 700;
+                font-size: 3.6rem;
+            }
+        }
+
+        &_list {
+            margin-top: 3rem;
+            padding: 3rem 0 1rem;
+            overflow-y: auto;
+            overflow-x: hidden;
+
+            &-item {
+                padding-left: 3.5rem;
+                color : $whiteColor;
+                line-height: 5.5rem;
+
+                span {
+                    font-size: 2.5rem;
+                    font-weight: 700;
+                }
+            }
+        }
+        &_account {
+            margin-top: 5rem;
+            display: flex;
+            align-items: center;
+            font-size: 3.5rem;
+            justify-content: space-around;
+            &-item {
+                color: $whiteColor!important;
+                
+            }
```

```suggestion
            &-item {
                color: $whiteColor!important;
            }
```
Code nhớ check lại cho cẩn thận nhé